integration/test/test_plugin_static_policy.py

```diff
@@ -70,7 +70,7 @@ def skip_unless_energy_agent():
         agent = getSystemConfigAgent()
     except BaseException as ex:
         return unittest.skip('geopmadmin check failed: {}'.format(ex))
-    if agent not in ['frequency_map', 'energy_efficient']:
```

Comment:
Lost the "not" qualifier

Reply:
:o! Thanks for catching that! Will fix!

Reply:
Just to clarify, do you mean:
- "skip unless rapl agent" or...
- "skip if rapl agent" ?